### ISSUE TITLE
[RA] Add faint player color to submerged units.

### ISSUE THREAD

**Issue by SoScared** [OpenRA/OpenRA#12237]
Mostly helpful for spectators. If the "shadow" or maybe parts of it would show off some color when submerged it would be easier to tell which unit belongs to who.

**Comment by jaZzKCS** [comment]
Yes indeed, maybe add the player's colour as a shadow to the unit.

**Comment by MustaphaTR** [comment]
:+1: , should be easy to implement. I'll take a look what i can do.

**Comment by MustaphaTR** [comment]
I changed them to use normal cloak palette. Not sure if this is better or not. If not i have another idea.
![subs using normal cloak palette](https://cloud.githubusercontent.com/assets/7933210/19415979/b83ebf5a-9382-11e6-90a6-2596fa835250.png)

**Comment by pchote** [comment]
Just color tint the submerged palette.
